### PR TITLE
retrieved data through the XHR object for the new evaluation page

### PR DESCRIPTION
**Pull request by aircraft721**

### CODE REVIEW

File: start-from-zero/Javascript/src/data/storeData.js [hunk 1]
```diff
@@ -42,6 +42,6 @@ windowObjectWrap.FormData = function(){
         const retrieveObject = localStorage.getItem('localArr');
         
         let parsedObject = JSON.parse(retrieveObject);    
-        console.log(parsedObject);
+        //console.log(parsedObject);
```
Comment (softvision-alexcarcea) [inline start-from-zero/Javascript/src/data/storeData.js]: This shall go away, yes?

File: start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js [hunk 2]
```diff
@@ -1,150 +1,162 @@
...
-                        ${SelectOption(
-                            windowObjectWrap.FieldsetData()
-                        )}
-                        <div class="button-wrap">
-                            <input type="button" id='btn' class='submit-button' value='Submit'>
+var xhr = new XMLHttpRequest();
+xhr.open('GET','../../data/data.js',true);
+xhr.onload = function(){
+    if(this.status == 200){
+        windowObjectWrap.NewEvaluationPage = function(options) {
```
Comment (softvision-alexcarcea) [inline start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js]: Any particular reason why this is deferred until the HTTP request is loaded?

File: start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js [hunk 3]
```diff
@@ -1,150 +1,162 @@
...
-    return input.join('');
+        }
+        
+        const SelectOption = function (options = {}) {
+            return `
+                ${TechnicalAreaPicker({ fieldsetObj: options.data })}
+            `;
+        };
+    }else{
+        console.log('something went wrong');
```
Comment (softvision-alexcarcea) [inline start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js]: Fun fact: there are other console APIs that are better suited for error conditions. Can you guess what they might be? :stuck_out_tongue:

**Review by softvision-alexcarcea — APPROVED** [review]

File: start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js (hunk 3 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js]: console.error 😛

File: start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js (hunk 2 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js]: No reason at all. I was just experimenting. I've just discovered now the differences between onload and onreadystatechange -> https://stackoverflow.com/a/19247992/7095201

File: start-from-zero/Javascript/src/data/storeData.js (hunk 1 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/data/storeData.js]: I have failed you :(

**Pull request merged**